RR_manuscript/Draft_RR.Rmd

```diff
@@ -1206,65 +1218,76 @@ In this non-preregistered model, the meta-analytic intercept estimate is between
 This suggests that the acoustic measures of typical word combinations are 0.01 standard units lower to 0.04 standard units higher than the measures of atypical word combinations, at 95% confidence.
 Relative to the preregistered model, this model suggests a somewhat positive effect of typicality (although small negative effects are also possible).
 
+<!-- TR: The meta analytical estimate is basically meaningless, given that the scale is not interpretable and artificial negative and positive estimates cancel each other out. I wonder whether we should take this out --> 
+
 Figure \@ref(fig:plot-meta1) illustrates the individual intercepts for critical typicality coefficients across models and teams, sorted in ascending order based on their mean.
-Given the nature and wide variety of acoustic operationalizations, there is not always a natural interpretation of the scale, but in most cases a positive effect corresponds to typical word combinations eliciting higher acoustic values (e.g. longer duration, higher f0, etc.), as mentioned above.
-When looking at the raw estimates and their variance (grey triangles and lines), it is striking how much teams' estimates differed. 
+Given the nature and wide variety of acoustic operationalizations, there is no natural interpretation of the scale, so we cannot interpret the direction of estimates.
+When looking at the raw estimates and their variance (grey triangles and lines), it is striking how much estimates differed. 
 Estimates ranged from `r round(min(msa_models$estimate),2)` to `r round(max(msa_models$estimate),2)` standard units. 
+We will interpret these standard units relative to phonetic measures in the discussion section, but to provide an intuition about the absolute scale of variation, we can express them on a standardized scale that does not require domain expertise: The IQ scale [@kamphaus2019clinical] which is defined by a standard deviation of 15. 
+The observed variability of estimates between teams corresponds to variability in IQ measures up to 14.9 IQ points.
```

Comment:
I don't think this is helpful. We need to give an example from phonetics, not other field.